Package.swift

```diff
@@ -33,7 +33,7 @@ let package = Package(
   ],
 
   dependencies: [
-    .package(url: "https://github.com/apple/swift-syntax.git", from: "510.0.1"),
+    .package(url: "https://github.com/apple/swift-syntax.git", branch: "main"),
```

Comment:
This change would need to be reverted for a commit when tagging a release, right?

Reply:
I did not intend to include that in the PR. Will remove.

Reply:
Removed.